lib/status/status.go

```diff
@@ -402,6 +407,26 @@ func diskSpaceProbeErrorDetail(p pb.Probe) (string, error) {
 	return data.FailureMessage(), nil
 }
 
+// String returns a textual representation of this system status
+func (r SystemStatus) String() string {
+	switch pb.SystemStatus_Type(r) {
+	case pb.SystemStatus_Running:
+		return "running"
+	case pb.SystemStatus_Degraded:
+		return "degraded"
+	default: // pb.SystemStatus_Unknown:
+		return "unknown"
```

Comment:
May not be necessary here, but I would consider separating the default -> "unknown" and SystemStatus_Unknown -> "unknown". The basic idea is, if a new SystemStatus code get's added, but this switch statement isn't updated, that the bug becomes visible and easier to locate.